rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,74 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Project completion stage RFC
+
+## Summary
+
+This RFC proposes a process of releasing a contribution, or project, only after it has been completed by the DCP in order to maximize the value to the data consumer through reducing user confusion and minimizing errors in the final product.
+
+## Author(s)
+
+ [Jason Hilton](mailto:jahilton@stanford.edu)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Definitions
+
+A contribution, or project, is considered to be **version-complete** when the contributor confirms (by way of the wrangler) that they have provided all of the metadata and raw data that they plan to submit at the current time, the wrangler submits those to ingest, and the DCP has delivered every possible product, given current DCP capabilities.
```

Comment:
I find project confusing because of the existing concept of BioProject.  Essentially related to a  grant or research group.

I find "dataset' by itself confusing, as it is very general and lot of people think they know what it means (GA4GH 1.0 horror story available on request).

I believe "experiement' or "experiment dataset" is much clearer and maps to how people think about science.

We also need to think how this terminology maps to primary and secondary analysis.

Maybe the whole mess is an experiment and with primary results are an experiment dataset, etc.

We really need to turn @justincc terminology table into an RFC.

Reply:
Depending on which archive/resource you look at, project, experiment, study and dataset are used to represent overlapping concepts. We picked project because the concept of the logical overarching set of data needed a label. If it is useful to change it we can.

I personally dislike experiment as in my head that represents the individual unit of work and output rather than the collection

That said if we get too concerned with avoiding terms that are understood in different contexts to mean different things we might as well pick a completely arbitrary word like banana and just give up trying to pick a word which is close in meaning already

The most important issue here is that we define what we mean when we use these words rather than necessarily change words (though if there is a consensus for a different term for that top-level grouping to project I won't stand in its way)

Reply:
>That said if we get too concerned with avoiding terms that are understood in different contexts to mean different things we might as well pick a completely arbitrary word like banana and just give up trying to pick a word which is close in meaning already.

Exact.

One idea is to namespace the word, for example "HCA Submission". We might also just accept that there will be some training about what a given word means in the HCA context.

Reply:
Adding HCA as a prefix to help understanding and provide context is a good idea.

That said, we need to be careful not to overload the word submission.

A submission is an event where a user interacts with the ingest system. A project might be made up of multiple submission events. So I think we need to pick a different word to that

Reply:
> The most important issue here is that we define what we mean when we use these words rather than necessarily change words

The rigorous definition of terminology is the key, otherwise, people make assumptions based on their experience.

Reply:
There's some inconsistency on whether a 'contribution' or 'submission' is provided to the DCP by data contributors/submitters (convo on the DataOps Charter). I have made this RFC internally consistent (submission/submitter for now and will update if the Charter convo goes the other way).
To me, _project_ has a very clear definition at the DCP & HCA. It is the primary organization unit that comes in to the DCP and that goes out to the user (click 'explore' on the portal and 'projects' is in the url, it's a tab label, click a _project_ title and you get to explore _project_, or select _project, or download the _project_ metadata).
I do agree that adding 'HCA' is a fabulous (and very necessary) idea - hopefully that distinguishes it from how other ~projects~ groups define 'project'.
UPDATED - Hopefully the included definitions and more consistent use convey my meaning better now, but let me know if there's still uncertainty - thanks to all for this!

Reply:
Does a project include all analysis generated by the DCP?

Reply:
@justincc Yes. I updated the definitions in c118d05.
I'll resolve this conversation, but please comment on the revisions if there's still questions/comments.